pushmsixscript/docker.d/init_worker.sh

```diff
@@ -10,10 +10,24 @@ test_var_set() {
   fi
 }
 
+export REQUEST_TIMEOUT_SECONDS=30
+export LOGIN_URL=https://login.microsoftonline.com
+export TOKEN_RESOURCE=https://manage.devcenter.microsoft.com
+export STORE_URL=https://manage.devcenter.microsoft.com/v1.0/my/applications/
+export MOCK_APPLICATION_ID=MOCK-APP-ID
+export BETA_APPLICATION_ID=9NZW26FRNDLN
+export RELEASE_APPLICATION_ID=9NZVDKPMR9RD
```

Comment:
This placement makes sense. (We may end up moving some of these into the `if` blocks below if we ever ship more than 1 product, but that may never happen.)